lib/registrant.rb

```diff
@@ -0,0 +1,23 @@
+class Registrant
+  attr_reader :name, :age, :permit, :license_data
```

Comment:
Don't need a `:permit` in the attr_reader since that is a method that's never used. We wrote the `permit?` method to act as the attr_reader.